pkgs/development/python-modules/jc/default.nix

```diff
@@ -9,12 +9,14 @@
 
 buildPythonPackage rec {
   pname = "jc";
-  version = "1.10.7";
+  version = "1.10.10";
   disabled = isPy27;
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "198vsnh6j0nv9d7msnvw6qr1bzf0nffjsz7clm11bs7fh3ri3qxp";
+  src = fetchFromGitHub {
+    owner = "kellyjonbrazil";
+    repo = "jc";
+    rev = "2c1935115de03a28fe92674b04b2542b0eccb8fe";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
Didn't know you could use tags in rev, neat!